examples/critical-example/README.rst

```diff
@@ -0,0 +1,62 @@
+Run Example
+===========
+
+1. Run penthouse container on port 3000.
+   Check whether it is running:
+
+.. code-block:: text
+
+    http://0.0.0.0:3000/?url=https://www.moccu.com/&css=https://www.moccu.com/static/css/styles.css
```

Comment:
Do you want to use moccu-urls for the example call?

Reply:
Any other ideas for working example?